### PR TITLE
feat: add context menu dropdown

### PR DESCRIPTION
**Pull request by Sebastien-Ahkrin**
close: https://github.com/zakodium-oss/analysis-ui-components/issues/260

### CODE REVIEW

**Comment by targos** [comment]
Can you open it aligned on the top left corner (like the native context menus)?

**Comment by Sebastien-Ahkrin** [comment]
> Ping @Sebastien-Ahkrin 

Ups ! I'm now working on it to make it good!

File: stories/components/dropdown.stories.tsx [hunk 1]
```diff
@@ -52,12 +52,41 @@ export function Dropdown() {
...
+    <DropdownMenu trigger="contextMenu" onSelect={() => {}} options={options}>
+      <div
+        style={{
+          height: '500px',
+          width: '500px',
+          border: '1px solid black',
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
```
Comment (targos) [inline stories/components/dropdown.stories.tsx:82]:
```suggestion
        }}
        tabIndex={1}
```

**Comment by targos** [comment]
Please fix the eslint error, that's what I'm waiting for :)

File: src/components/menu/DropdownMenu.tsx [hunk 2]
```diff
@@ -24,18 +25,72 @@ interface DropdownMenuClickProps<T> extends DropdownMenuBaseProps<T> {
...
+    styles,
+    attributes,
+  } = useContextMenuPlacement(otherProps.placement || 'right-start');
+
+  const ref = useRef<HTMLDivElement>(null);
+  useOnClickOutside(ref, closePopperElement);
+
+  return (
+    <div
+      id="test"
```
Comment (targos) [inline src/components/menu/DropdownMenu.tsx:69]: ?

**Pull request merged**